examples/vue-kitchen-sink/src/stories/components/button.stories.js

```diff
@@ -5,10 +5,19 @@ export default {
   component: MyButton,
 };
 
-export const Rounded = () => ({
+export const Rounded = (args) => ({
   components: { MyButton },
-  template: '<my-button :rounded="true">A Button with rounded edges</my-button>',
+  template: '<my-button :color="color" :rounded="rounded">A Button with rounded edges</my-button>',
+  data() {
+    return args;
```

Comment:
@shilman Does the `args` value ever change after the initial story render? Cause, if so, I'm not sure this will be properly reactive. A.k.a Vue itself won't trigger a rerender when the value changes... Unless the whole story gets completely rerendered (as in, the vue context gets remounted by the react it's build on) any time the args change?

Reply:
Yeah, the story gets re-rendered every time the args change.